layouter/static/layouter/css/font-awesome.min.css

```diff
@@ -0,0 +1,4 @@
+/*!
```

Comment:
I think we should only use the css rules which are actually needed. So we should keep the license, the font-fact and the css rules for the 3 icons. I would like to avoid unnecessary CSS in the plugin.